napari/_qt/_qapp_model/_tests/test_qapp_model_menus.py

```diff
@@ -0,0 +1,12 @@
+import pytest
+
+from napari._app import app, constants
+from napari._qt._qapp_model import build_qmodel_menu
+
+
+@pytest.mark.parametrize('menu_id', list(constants.MenuId))
+def test_build_qmodel_menu(qtbot, menu_id):
+    """Test that we can build qmenus for all registered menu IDs"""
+    menu = build_qmodel_menu(menu_id)
+    qtbot.addWidget(menu)
+    assert len(menu.actions()) >= len(app.menus.get_menu(menu_id))
```

Comment:
why longer, not equal?

Reply:
ah, because of submenus.  the point here is just to test that it's populated, and to make sure that nothing upstream changes this here.  The actual population of a qmenu from an app model is well tested in app-model